adapters/handlers/rest/configure_api.go

```diff
@@ -298,14 +298,49 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup, is
 	explorer.SetSchemaGetter(schemaManager)
 	appState.Modules.SetSchemaGetter(schemaManager)
 
-	err = vectorRepo.WaitForStartup(ctx)
+	// TODO-RAFT START
+
+	candidateList, err := resolveRaftAddresses(appState)
 	if err != nil {
 		appState.Logger.
+			WithField("action", "startup").
 			WithError(err).
+			Fatal("cannot resolve raft addresses using memberlist")
```

Comment:
Should we improve this and dump the list of addresses/nodes we tried to resolve ?